CHANGELOG.md

```diff
@@ -13,6 +13,45 @@ After this, `Decidim::Proposals::ProposalEndorsement` and the corresponding coun
 
 ### Added
 
+- **decidim-core**: Allow users to register with a preferred language. [\#5789](https://github.com/decidim/decidim/pull/5789)
+- **decidim-proposals**: Add images to proposal cards [\#5640](https://github.com/decidim/decidim/pull/5640)
```

Comment:
I guess many of these entries are from an automated merge that must be manually solved

Reply:
Damn yes